pkgs/applications/networking/mumble/default.nix

```diff
@@ -118,7 +116,6 @@ let
         "-D server=OFF"
         "-D bundled-celt=ON"
         "-D bundled-opus=OFF"
-        "-D bundled-speex=OFF"
```

Comment:
Hm. They still have a build option for it and the code also has hints on speex as well. https://github.com/mumble-voip/mumble/blob/master/docs/dev/build-instructions/cmake_options.md#bundled-speex

Reply:
That is really strange. I found [this pull request that explicitly removes support for the speed codec](https://github.com/mumble-voip/mumble/pull/5869). Also the [3rd-party folder](https://github.com/mumble-voip/mumble/tree/master/3rdparty) does not contain the speed codec anymore, but only some speex-dsp library.

Actually, they did drop support for the Speex codec, but kept the flag to mean usage of the speex-dsp for ... something. That is actually a different package in nix and a different dependency. :/

I have added that dependency as a new commit and added the option again.